wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -45,7 +37,11 @@ Other situations do not require a 24 hour wait. This gives other Beatmap Nominat
 + **Ensure that kudosu is awarded to all posts that deserve it.** Additionally consult the mapper before giving out kudosu to make sure there are not any reasons why the post does not need kudosu (e.g. the modder has received one already for a different post which you overlooked). If the modpost is too old to receive kudosu, consult the modder to make an additional post linking the original he receives kudosu for.
 + **You must not place any icons besides the star, bubble, bubble pop and heart.** Whilst you are able to set other icons, they are either deprecated and discouraged from use (flame, circle, drum, apple, piano) or reserved for use by the Quality Assurance Team (nuke). If you accidentally set an unwanted icon, contact a QAT immediately or overwrite the icon in case you wanted to post a bubble or heart.
 + **Do not nominate a beatmap without properly checking all difficulties of the beatmap all by yourself.** If you are unfamiliar with the game mode some of the difficulties are mapped in, ask fellow Nominators for help. Improper checking includes just running a modding tool before nominating or not even downloading the beatmap.
-+ **You can veto a beatmap set by placing a bubble pop icon or a problem stamp on the set that you disagree with being fit for being Ranked.** Placing a beatmap veto over any kind of quality issues that you think need to be addressed before the map can move anywhere is one of your main responsibilities. A proper discussion with attempting to reach a mutual agreement by both sides of the argument has to follow.
++ **You can veto a qualified beatmap set by posting the issues you think need to be addressed before reaching ranked, and then contacting a member of the QAT.** You must participate in the resulting discussion and attempt to reach a mutual agreement. The following conditions must be met before the veto is applied and the set is disqualified, however:
+  + The veto post must include a clear problem and reasoning for pausing the beatmap set's ranking process.
+  + The veto post must be at least 1 day old to allow for observation of test plays and additional feedback.
```

Comment:
just to clarify, does this mean it's impossible to veto a map when it's less than one day away from ranking? that seems wrong

Reply:
no that's meant to be an exception, think I may have forgotten to include that bit:
"/.../ at least 1 day old (unless there is less than 1 day left until rank) to allow for /.../".

Reply:
adjusted